doc-site/components/nve-switch.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: component
+---
+
+<CodeExamplePreview>
+
+```html
+<nve-switch></nve-switch>
+```
+
+</CodeExamplePreview>
+
+## Eksempler
+
+Med ikoner:
+<CodeExamplePreview>
+
+```html
+<nve-switch>
+  <nve-icon slot="officon" name="light_mode"></nve-icon>
+  <nve-icon slot="onicon" name="dark_mode"></nve-icon>
+</nve-switch>
+```
+
+</CodeExamplePreview>
+
+Label:
+<CodeExamplePreview>
+
+```html
+<nve-switch>
+  <div slot="label">Slå på</div>
+</nve-switch>
+```
+
+</CodeExamplePreview>
+
+Disabled:
```

Comment:
Det er lov å skrive en forklaring før kodeeksemplet, altså:) Noen liker å bli matet med teskje. 
Eksempel: "Bruk `disabled` for å deaktivere bryteren."